public-dashboard-client/src/viz-recidivism-rates/RecidivismRatesChart.js

```diff
@@ -136,7 +137,10 @@ export default function RecidivismRatesChart({ data, highlightedCohort }) {
                 }}
                 renderKey={(d) =>
                   // if it has a label, it's a line; if not, it's a point
-                  d.label || `${d.releaseCohort}-${d.followupYears}`
+                  d.label ||
+                  `${d.releaseCohort}-${d[DIMENSION_DATA_KEYS.race]}${
```

Comment:
I finally figured out what you mean by lines and points. They are the lines and points passed into the `XYFrame`.    
Maybe it would be more clear to add a `label` to each point on line 62 when they are created? Or in `prepareChartData`. That may be a little more clear how the points are going to be labeled and avoid this confusion. Just a thought.

Reply:
yes great suggestion!